### PR TITLE
distinguish between request and response pdu

### PR DESCRIPTION
**Pull request by lamphamsy**
It's to distinguish between request and response pdu

### CODE REVIEW

File: lib/kinetic.js [hunk 1]
```diff
@@ -1632,14 +1632,13 @@ function parsePDU(socket, callback) {
...
-                if (pdu.getMessageType() !== ops.NOOP) {
+                if (pdu.getCommand().status &&
+                    pdu.getMessageType() !== ops.NOOP) {
                     error = (pdu.getStatusCode() === errors.SUCCESS) ? null :
                         pdu.getErrorMessage();
                 }
                 return callback(error, pdu);
             }
-        } else {
-            return callback({ noHeader: true });
```
Comment (koolfunky) [inline lib/kinetic.js]: Why did you remove this ?

Comment (lamphamsy) [reply lib/kinetic.js]: Sorry, I was wrong :). It should be there.

**Review by koolfunky — APPROVED** [review]
